.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 *.iml
+*.swp
```

Comment:
@manonthemat  the default file for the global ignore file is actually `$HOME/.config/git/ignore`. There's no need to set it to something else as in the link by @gergoerdosi